contrib/ansible/openshift/pbench_register.yml

```diff
@@ -19,19 +19,23 @@
         group: root
         mode: 0755
     - name: add masters to the file
-      shell: echo -e {{ item.1 }} master {{ item.0 }} svt_master_{{ item.0}} >> {{ file }}
+      shell: echo -e {{ item.1 }} master {{ item.0 }} svt_master >> {{ file }}
```

Comment:
Let's just pass "`svt_`", since parameter 2 already has "master" (or "node", "etcd", "lb").  Then in the script, we can add that prefix.